packages/pluggableWidgets/pie-doughnut-chart-native/src/__tests__/PieDoughnutChart.spec.tsx

```diff
@@ -0,0 +1,340 @@
+import { EditableValueBuilder, ListAttributeValueBuilder, ListValueBuilder } from "@mendix/piw-utils-internal";
+import { act, fireEvent, render, Thenable } from "react-native-testing-library";
+import { ReactTestInstance } from "react-test-renderer";
+import { ObjectItem } from "mendix";
+import { View } from "react-native";
+import { createElement } from "react";
+import { Big } from "big.js";
+
+import { PieDoughnutChartProps, SeriesType } from "../../typings/PieDoughnutChartProps";
+import { ChartStyle, defaultStyle } from "../ui/Styles";
+import { PieDoughnutChart } from "../PieDoughnutChart";
+
+jest.mock("victory-native", () => ({
+    VictoryPie: jest.fn(({ children, ...rest }) => (
+        <View testID="mock" {...rest}>
+            {children}
+        </View>
+    ))
+}));
+
+describe("PieDoughnutChart", () => {
+    describe("configuration", () => {
+        it("basic static props", () => {
+            const { getByTestId, toJSON } = render(<PieDoughnutChart {...createProps()} />);
```

Comment:
Do you think the code would be simpler if we have a defaultProps object instead of the createProps function? Just like we have in the native badge tests for example

Reply:
At first I had a default object and then if I needed to override any properties I would spread them onto the component after first spreading the default props. This got messy so I opted for the function which accepts arguments that can override defaults.

If you can think of an example that would make things simpler please suggest one, otherwise I think this is good.

Reply:
I think the approach is very similar, the function is also fine by me